frontend/apps/web/app/[account]/transformers/components/UserDefinedTransformersTable/columns.tsx

```diff
@@ -52,7 +52,14 @@ export function getUserDefinedTransformerColumns(
         return (
           <div className="flex space-x-2">
             <span className="max-w-[500px] truncate font-medium">
-              {row.original.name}
+               <div>
+                  <NextLink
+                    className="hover:underline"
+                    href={`/${account?.name}/transformers/${transformer.id}`}
```

Comment:
Looks like there are some build errors here with the `account` not being present.

Check out how we are adding the `accountName` to the columns component here: https://github.com/nucleuscloud/neosync/blob/main/frontend/apps/web/app/(mgmt)/%5Baccount%5D/jobs/components/DataTable/columns.tsx#L46

So the `accountName` would get added as a prop and be passed down.